thirdeye/thirdeye-pinot/src/main/resources/migrations.xml

```diff
@@ -230,4 +230,21 @@
       </column>
     </addColumn>
   </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+
+  <changeSet id="7" author="nprabhak">
+    <addColumn tableName="anomaly_functions">
+      <column name="window_delay_unit" type="varchar(63)">
+        <constraints nullable="false"/>
```

Comment:
existing rows will break on `nullable="false"` - see if hibernate supports this in migrations, otherwise do one of following:
1. Add nullable="false", but also include a non-null default value. Remove default value in a following migration.
2. Use nullable="true", run an update adding a non-null default value for existing rows, then set nullable="false"